### PR TITLE
made shutdown timer customizable

### PR DESCRIPTION
**Pull request by andrei-mu**

### CODE REVIEW

File: include/sentry.h [hunk 1]
```diff
@@ -1006,6 +1006,13 @@ SENTRY_API void sentry_options_set_database_pathw(
 SENTRY_API void sentry_options_set_system_crash_reporter_enabled(
     sentry_options_t *opts, int enabled);
 
+/**
+ * Sets the maximum time to wait for the asynchronous tasks to end on shutdown,
+ * before attempting a forced termination.
+ */
+SENTRY_API void sentry_options_set_shutdown_timeout(
```
Comment (Swatinem) [inline include/sentry.h:1013]: a corresponding `get` would be nice as well

**Review by Swatinem — APPROVED** [review]

**Review by Swatinem — APPROVED** [review]
Sorry for the turnaround here, forgot to mention this in the last review: It would be nice to document that all the times are in milliseconds.

File: include/sentry.h (hunk 1 above)
Comment (andrei-mu) [reply include/sentry.h:1013]: I've added the getter as well, please review.

**Comment by andrei-mu** [comment]
> Sorry for the turnaround here, forgot to mention this in the last review: It would be nice to document that all the times are in milliseconds.

this is done as well.

**Review by Swatinem — APPROVED** [review]

**Pull request merged**